client/state/selectors/get-my-sites-menu-item-types.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import { endsWith, filter, find, get } from 'lodash';
+ // TODO: Calling translate in selectors is not ideal
```

Comment:
Could the translated items be stashed in an array perhaps in the component to get around this?  Either way its not the most ideal setup.